docs/reference/architecture/api.rst

```diff
@@ -54,36 +60,46 @@ In order to activate the API's, you'll also need to add this to your routing:
         type:         rest
         prefix:       /api/ecommerce
         resource:     "@SonataProductBundle/Resources/config/routing/api.xml"
+        # or for nelmio/api-doc-bundle v3
+        #resource:     "@SonataProductBundle/Resources/config/routing/api_nelmio_v3.xml"
```

Comment:
Documentation is mostly read by people that are currently installing the project. I'd recommend documenting the most modern way of doing things and only that one.